File: SDKs/Aspose.Pdf_Cloud_SDK_For_Java/src/main/java/com/aspose/pdf/model/TextState.java
package com.aspose.pdf.model;

import com.aspose.pdf.model.Color;
import com.aspose.pdf.model.FontStyles;
public class TextState {
  private Float FontSize = null;
  private String Font = null;
  private Color ForegroundColor = null;
  private Color BackgroundColor = null;
  private FontStyles FontStyle = null;
  /**
	 * getFontSize
	 * Gets Float
	 * @return FontSize
	 */
  public Float getFontSize() {
    return FontSize;
  }

	/**
	 * setFontSize
	 * Sets Float
	 * @param FontSize Float
	 */
  public void setFontSize(Float FontSize) {
    this.FontSize = FontSize;
  }

  /**
	 * getFont
	 * Gets String
	 * @return Font
	 */
  public String getFont() {
    return Font;
  }

	/**
	 * setFont
	 * Sets String
	 * @param Font String
	 */
  public void setFont(String Font) {
    this.Font = Font;
  }

  /**
	 * getForegroundColor
	 * Gets Color
	 * @return ForegroundColor
	 */
  public Color getForegroundColor() {
    return ForegroundColor;
  }

	/**
	 * setForegroundColor
	 * Sets Color
	 * @param ForegroundColor Color
	 */
  public void setForegroundColor(Color ForegroundColor) {
    this.ForegroundColor = ForegroundColor;
  }

  /**
	 * getBackgroundColor
	 * Gets Color
	 * @return BackgroundColor
	 */
  public Color getBackgroundColor() {
    return BackgroundColor;
  }

	/**
	 * setBackgroundColor
	 * Sets Color
	 * @param BackgroundColor Color
	 */
  public void setBackgroundColor(Color BackgroundColor) {
    this.BackgroundColor = BackgroundColor;
  }

  /**
	 * getFontStyle
	 * Gets FontStyles
	 * @return FontStyle
	 */
  public FontStyles getFontStyle() {
    return FontStyle;
  }

	/**
	 * setFontStyle
	 * Sets FontStyles
	 * @param FontStyle FontStyles
	 */
  public void setFontStyle(FontStyles FontStyle) {
    this.FontStyle = FontStyle;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TextState {\n");
    sb.append("  FontSize: ").append(FontSize).append("\n");
    sb.append("  Font: ").append(Font).append("\n");
    sb.append("  ForegroundColor: ").append(ForegroundColor).append("\n");
    sb.append("  BackgroundColor: ").append(BackgroundColor).append("\n");
    sb.append("  FontStyle: ").append(FontStyle).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Pdf_Cloud_SDK_for_.NET/src/Com/Aspose/PDF/Model/TextState.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class TextState {
    public float? FontSize { get; set; }

    public string Font { get; set; }

    public Color ForegroundColor { get; set; }

    public Color BackgroundColor { get; set; }

    public FontStyles FontStyle { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TextState {\n");
      sb.Append("  FontSize: ").Append(FontSize).Append("\n");
      sb.Append("  Font: ").Append(Font).Append("\n");
      sb.Append("  ForegroundColor: ").Append(ForegroundColor).Append("\n");
      sb.Append("  BackgroundColor: ").Append(BackgroundColor).Append("\n");
      sb.Append("  FontStyle: ").Append(FontStyle).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
